website/docs/d/ad_access_credentials.html.md

```diff
@@ -53,7 +53,7 @@ The following arguments are supported:
 
 * `namespace` - (Optional) The namespace of the target resource.
   The value should not contain leading or trailing forward slashes.
-  The `namespace` is always relative to the provider's configured [namespace](/docs/providers/vault#namespace).
+  The `namespace` is always relative to the provider's configured [namespace](/docs/providers/vault/index.html#namespace).
```

Comment:
I am not actually certain that this will work. Maybe we should link to the registry documentation instead of the relative link?

Reply:
I believe this should work. For what it's worth I'm seeing examples in [other Providers for this pattern](https://github.com/hashicorp/terraform-provider-aws/blob/main/website/docs/r/chime_voice_connector.html.markdown?plain=1#L33) as well as using the [registry documentation pattern](https://github.com/hashicorp/terraform-provider-aws/blob/main/website/docs/r/directory_service_region.html.markdown?plain=1#L127), so I don't think we can go wrong with either :)